CHANGELOG.rst

```diff
@@ -2,10 +2,20 @@
 History
 =======
 
-0.16.0 (unreleased)
+0.16.0 (2023-02-09)
```

Comment:
Minor thing:

Should we already add a new version `0.16.0 (unreleased)` or it should be added after the current release?

Reply:
This will be the 0.16.0 release.

For PRs in master we probably should add a `0.17.0 (UNRELEASED)` section.
Or merges can wrongly add to the `0.16.0` section.